sapientml_core/templates/other_templates/confusion_matrix.py.jinja

```diff
@@ -0,0 +1,6 @@
+{% if pipeline.task.task_type == 'classification' %}
+# Confusion Matrix
+from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay
+cm = confusion_matrix(target_test, y_pred)
+ConfusionMatrixDisplay.from_predictions(target_test, y_pred)
```

Comment:
We should consider cases of multiple target columns. How about the following implementation?
```
from sklearn.metrics import ConfusionMatrixDisplay
{% if pipeline.task.target_columns|length == 1 %}
ConfusionMatrixDisplay.from_predictions(target_test, y_pred)
{% else %}
for i, column in enumerate(target_test.columns):
    disp = ConfusionMatrixDisplay.from_predictions(target_test[column], y_pred[:, i])
    disp.ax_.set_title(column)
{% endif %}
```